bin/osg-notify

```diff
@@ -185,7 +191,11 @@ def main():
         if args.fqdn_filter:
             results = topology_utils.get_resource_contacts_by_fqdn(args)
         else:
-            results = topology_utils.get_resource_contacts(args)
+            try:
+                # throws IOError for incorrect password
+                results = topology_utils.get_resource_contacts(args)
+            except IOError:
+                print("ERROR: incorrect password.")
```

Comment:
Since we were seeing something like:

```
IOError: \[Errno 22] Invalid argument
```

You should check that the `errno` of the exception is 22, then `sys.exit` with your error message (we don't want the script to continue). If it's another errno, then you should raise the exception

Reply:
Do you have a traceback for this exception? I don't see any mention of IOError in topology_utils; if something in topology_utils is raising (or propagating) an IOError, then that should be at least documented *in topology_utils*.

Although what I would do is catch the IOError in topology_utils, then make an "IncorrectPassword" exception and raise that instead.

Reply:
Traceback here: https://opensciencegrid.atlassian.net/browse/SOFTWARE-3914?focusedCommentId=356097